app/views/posts/_post.html.erb

```diff
@@ -30,16 +30,16 @@
     <div class="mb-3">
       <div class="row">
         <div class="col">
-          <div class="border border-dark" style='width: 200px; height: 20px; border-style: dashed !important'>
-            <!-- Like button -->
-          </div>
+          <%= button_to post_like_path(post), method: :patch, class: "btn btn-primary"  do %>
+            <%= "#{post.likes.where(user: current_user).exists? ? "👎 Unlike" : "👍 Like"}" %>
```

Comment:
Clever use of the emojis! I don't know which problems came with the Icons but thats good to switch to use something to address it.